rcsb/utils/dictionary/DictMethodEntryHelper.py

```diff
@@ -623,24 +624,25 @@ def consolidateAccessionDetails(self, dataContainer, catName, **kwargs):
         try:
             logger.debug("Starting with  %r %r %r", dataContainer.getName(), catName, kwargs)
             #
+            internalEntryId = None
             # Add missing pdbx_database_status for MA or AF models (if absent in mmCIF file)
             cName = "pdbx_database_status"
             if not dataContainer.exists(cName):
                 if dataContainer.exists("entry") and dataContainer.exists("ma_data"):
                     dObj = dataContainer.getObj("entry")
-                    entryId = dObj.getValue("id", 0)
+                    internalEntryId = dObj.getValue("id", 0)
                     compModelSourceId = None
-                    if entryId.upper().startswith("MA_") or entryId.upper().startswith("MA-") or entryId.upper().startswith("AF_"):
-                        compModelSourceId = self.__mcP.getCompModelData(entryId)["sourceId"]
-                        logger.debug("compModelSourceId %r for entryId %s", compModelSourceId, entryId)
+                    if internalEntryId.upper().startswith("MA_") or internalEntryId.upper().startswith("MA-") or internalEntryId.upper().startswith("AF_"):
```

Comment:
@piehld Do we need the if statement on line 635?

Reply:
Yes, that's what was necessary to make the pipeline work for the modified cif files

Reply:
(it's an updated version of line 629 previously)

Reply:
Previously, we were applying the workaround for MA models only and so this conditionality was required to identify MA models. At present, we are applying this workaround for all CSMs and I am not sure if we need this given that we are already checking for the presence of `entry` and `ma_data` tables.

Reply:
This is a fix for how we are currently retaining CSM files internally, in which the pdbx_database_status is retained if present. I think what we end up doing here exactly will depend on what we decide to do with that field in the model files themselves (https://github.com/rcsb/py-rcsb_utils_insilico3d/pull/19#issuecomment-1175766146). 

But I agree ultimately it would make sense to just skip that conditional.

Reply:
Since the AF models do contain `pdbx_database_status.recvd_initial_deposition_date`, I was thinking we may end up wanting to introduce a workaround based on CSM type (i.e., MA or AF), since we would want to retain the AF deposition date, whereas for MA we would set it to be `?`. So that's why I felt like we might as well keep this logic in here until we come to a decision on what to do with `pdbx_database_status`

Reply:
Ok, thanks. Let's leave it as is for now.